package.json

```
@@ -21,7 +21,7 @@
     "start": "npm run storybook:dev",
     "analyze": "cem analyze",
     "commit": "commit",
-    "build": "node scripts/build.js && npm run analyze",
+    "build": "npm run analyze && npm run generate-react-exports && node scripts/build.js && npm run analyze",
```

We can generate react exports as last step here, right? 
```suggestion
    "build": "node scripts/build.js && npm run analyze && npm run generate-react-exports",
```

---

Ah, no, build needs `src/baklava-react.ts` file. But I didn’t like to run analyze step two times here 😊

---

yes, build needs `src/baklava-react.ts`. maybe we can generate `baklava-react.js` and add it to`dist` as manually after build.

---

Source of problem here is that build script deletes `dist` folder before generating new build. I think it will be wise to extract this step from build script and clean dist folder in `npm run build` command. Like:

```
"build": "del-cli dist/ && npm run analyze && npm run generate-react-exports && node scripts/build.js"
```

You can add `del-cli` as dev dependency (we were already use `del` package).